### PR TITLE
Adding Caption

### PR DESCRIPTION
**Pull request by preetiadvani**
Comment

### CODE REVIEW

**Review by JCpoz123 — APPROVED** [review]
reviewed

**Comment by preetiadvani** [comment]
.

File: _slides/08-preetiadvani.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Caption for Preeti Advani
```
Comment (preetiadvani) [inline _slides/08-preetiadvani.md:6]: Wonderful!

**Review by preetiadvani — COMMENTED** [review]
!!!

**Pull request merged**